.gitattributes

```diff
@@ -1,2 +1,5 @@
 *.cs text eol=lf
 *.vb text eol=lf
+
+# Prevent regression-test.ps1 to emit "warning: CRLF will be replaced by LF in analyzers" for each json file
+*.json text eol=crlf
```

Comment:
I wonder, is there a possibility to create the files with LF, to avoid having LF for C#/VB.NET  and CRLF for json?

Reply:
That's a very good question

Reply:
Local repo will be still polluted with CRLF and the warning. To clear it:
* pull commits from this PR
* delete `expected/*.json`
* revert the deletion, git will recreate those local files with LF

I'll leave a message on our Slack about this once it's merged.